src/main/java/discord/TwoDee.java

```diff
@@ -28,13 +32,14 @@ public static void main(String[] args) {
             new DiscordApiBuilder().setToken(token).setAllIntentsExcept(Intent.GUILD_PRESENCES).setUserCacheEnabled(true).addListener(velen).login().thenAccept(api -> {
                         // Uncomment this line when a command is altered
                         // TODO do this a smarter way
-                        // velen.registerAllSlashCommands(api);
+                        velen.registerAllSlashCommands(api);
```

Comment:
I would comment this out so that the bot doesn't register a ton of global commands on startuo

Reply:
I don't think that's a necessary concern? I mean let them. there's like 10. Making sure to have things updated, over debugging why they didn't is my  concern (Because that happened to me while testing)

Reply:
It might make things weird when they get updated in rapid succession, and the code says to not call that method unless you actually mean to update the commands globally

Reply:
Fair. I guess it's a bigger problem, since you use the same token for testing and running.